pkg/controller/feature-gates.go

```diff
@@ -0,0 +1,59 @@
+package controller
```

Comment:
Probably have a dedicated package for this.  Will not be used for just controllers.

Reply:
done